src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -61,6 +61,23 @@ public DefaultServer(List<Type> typesToInclude)
             await builder.DefinePersistence(settings);
 
             builder.GetSettings().SetDefault("ScaleOut.UseSingleBrokerQueue", true);
+            builder.NotifyOnFailedMessage(failedMessage =>
```

Comment:
@timbussmann so would making it an "add new action" api meet your future requirements?  I want to avoid making it an event since that implies it can be added or removed at runtime